business_requirement_deliverable_report/__openerp__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# © 2016 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Deliverable Report Module',
+    'sumarry': 'Business Requirement Deliverable Report Module',
+    'version': '8.0.5.0.1',
+    'category': 'report',
+    'website': 'https://www.elico-corp.com',
+    'author': 'Elico Corp',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'depends': [
+        'business_requirement_deliverable',
+    ],
+    'data': [
+        'views/report_business_requirement.xml',
+        'views/report_business_requirement_deliverable.xml',
+        'views/report_business_requirement_deliverable_resource.xml',
+        'report/report.xml'
+    ],
+    'image': [
+        'static/img/bus_req_tree.png'
+        'static/img/bus_req_module_diag.png',
+        'static/img/bus_req_report1.png',
+        'static/img/bus_req_report2.png',
+        'static/img/bus_req_report3.png',
+    ],
```

Comment:
As per Pedro's comment, keep only what is specifically relevant for this module features. You should remove images that don't illustrate features implemented here.